backend/danswer/db/models.py

```diff
@@ -138,7 +143,7 @@ class User(SQLAlchemyBaseUserTableUUID, Base):
 
 
 class AccessToken(SQLAlchemyBaseAccessTokenTableUUID, Base):
-    pass
+    expiry_length: Mapped[int] = mapped_column(Integer, nullable=False)
```

Comment:
do we need to add this? Can we instead compute it on the fly in the `/me` endpoint?

Reply:
Certainly can treat the expiry length time in seconds (environment variable) as a constant and use that instead